ultralytics/data/explorer/explorer.py

```diff
@@ -162,7 +169,7 @@ def query(self, imgs=None, limit=25):
         embeds = torch.mean(torch.stack(embeds), 0).cpu().numpy() if len(embeds) > 1 else embeds[0].cpu().numpy()
         return self.table.search(embeds).limit(limit).to_arrow()
 
-    def sql_query(self, query, return_type='pandas'):
+    def sql_query(self, query: str, return_type: str = 'pandas') -> Union[DataFrame, Any, None]:
```

Comment:
Here return type should be Union of dataframe and pyarrow table too I think?

Reply:
The ci checks failed when I imported `pyarrow`, so I removed it. Also, `pyarrow.Table` shows as `Any` when added as a type hint, so I used this (thought about adding a comment but figured the docstring would suffice).